Dockerfile

```diff
@@ -24,9 +26,3 @@ LABEL name="konveyor/forklift-controller" \
       io.openshift.tags="operator,konveyor,forklift,controller" \
       io.openshift.min-cpu="100m" \
       io.openshift.min-memory="350Mi"
-
-COPY --from=builder /opt/app-root/src/manager /usr/local/bin/manager
-
-RUN microdnf -y install tar && microdnf clean all
```

Comment:
`Tar` is installed to support `oc cp` command needed to download memorycpu profiles and potentially the inventory sqlite3 DB for troubleshooting.  With this PR we can no longer do this.  The ubi-micro does not include a package manager so we'd need to switch back to ubi-minimal.

Reply:
I've created #326 to address that.